distributions/aws/test/e2e/fixtures/clients.py

```diff
@@ -119,3 +131,26 @@ def kfp_client(port_forward, host, client_namespace, session_cookie):
     ] = client_namespace  # needs to be set for list_experiments
 
     return client
+
+
+@pytest.fixture(scope="class")
```

Comment:
remove the fixture from these aws client methods because credentials can rotate because of test time and fixtures are initialized once

Reply:
makes sense